### PR TITLE
Remove extra quote for prometheus request

### PR DESCRIPTION
**Pull request by adam2k**
## Description

Remove extra `"` breaking the podImageTag prometheus request.  Add error handling for the prometheus requests  where it was missing.

## 🎟 Issue(s)

Resolves astronomer/issues#1400

## 🧪 Functional Testing

> List the functional testing steps to confirm this feature or fix.

## 📸 Screenshots

> Add screenshots to illustrate the validity of these changes.

## 📋 Checklist

- [x] Rebased from the main (or release if patching) branch (before testing)
- [x] Ran `yarn lint && yarn test` before taking out of draft
- [x] Added/updated applicable tests
- [x] Tested against [Astro-UI](https://github.com/astronomer/astro-ui/) and [Astro-CLI](https://github.com/astronomer/astro-cli/) clients.
- [ ] Communicated to/tagged owners of respective clients potentially impacted by these changes.
- [ ] Updated any related [documentation](https://github.com/astronomer/docs/)

### CODE REVIEW

File: src/lib/deployments/config/index.js [hunk 1]
```diff
@@ -688,8 +688,8 @@ export function airflowImageForVersion(version) {
  */
 export async function getPodImageTag(releaseName) {
   const res = await prometheusRequest(
-    `kube_pod_container_info{pod=~"${releaseName}-scheduler-.*[A-Za-z0-9-].*",container="scheduler"}"`
```
Comment (adam2k) [inline src/lib/deployments/config/index.js:691]: @andriisoldatenko this quote was causing most of the issues in Dev.  I also added error handling.

**Review by kushalmalani — APPROVED** [review]

File: src/lib/deployments/config/index.js [hunk 2]
```diff
@@ -688,8 +688,8 @@ export function airflowImageForVersion(version) {
...
-  );
+    `kube_pod_container_info{pod=~"${releaseName}-scheduler-.*[A-Za-z0-9-].*",container="scheduler"}`
+  ).catch(err => log.error(err));
```
Comment (andriisoldatenko) [inline src/lib/deployments/config/index.js:692]: does it prevent to block resolver?

Comment (adam2k) [reply src/lib/deployments/config/index.js:692]: yeah, I was able to test locally without any prometheus running and this works.  It's the same error handling in prod for `airflow_heartbeat`.

Comment (andriisoldatenko) [reply src/lib/deployments/config/index.js:692]: cool

**Review by andriisoldatenko — APPROVED** [review]

**Pull request merged**